.buildkite/dlk-pipeline.yml

```diff
@@ -62,4 +54,4 @@ steps:
     env:
       BUILDKITE_CLEAN_CHECKOUT: 'true'
       DEVICE_TEST_INPUT_PATH: './output/TestCodeGenerationAarch64'
-      PYTHONPATH: 'dlk/python/dlk/scripts/pylib'
+      PYTHONPATH: '.'
```

Comment:
Is this necessary? I think Python might look into current directory with empty PYTHONPATH

Reply:
No, It didn't work in the raspberry-pi when I checked. let me work on that in another issue. thx.

Reply:
In the dir,
```
$ DEVICE_TEST_INPUT_PATH=./output/TestCodeGenerationAarch64 python3 ./tests/device_tests/test_device.py
Traceback (most recent call last):
  File "./tests/device_tests/test_device.py", line 21, in <module>
    from blueoil.converter.nnlib import NNLib
ModuleNotFoundError: No module named 'blueoil'
```

Reply:
(however, in python3 interpreter on the same dir, it works...)